diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..578bc19
--- /dev/null
+++ ./.gitignore
@@ -0,0 +1 @@
+db_dir/
diff --git a/quotes_spider/spiders/quotes_spider.py b/quotes_spider/spiders/quotes_spider.py
index f9881bd..d7a9006 100644
--- ./quotes_spider/spiders/quotes_spider.py
+++ ./quotes_spider/spiders/quotes_spider.py
@@ -1,11 +1,11 @@
-from scrapy import Request, Spider
+from scrapy import Spider
 from scrapy.loader import ItemLoader
 from quotes_spider.items import QuotesSpiderItem
 
 class QuotesSpider(Spider):
     name = 'quotes'
 
-    start_urls = ['http://quotes.toscrape.com']
+    start_urls = ['https://rewrfsrewr.xyz/forum.php?mod=forumdisplay&fid=103&filter=typeid&typeid=481']
 
     def parse(self, response):
         quotes = response.xpath('//div[@class="quote"]')
